cluster/addons/fluentd-elasticsearch/fluentd-es-image/run.sh

```diff
@@ -20,4 +20,4 @@
 # For systems without journald
 mkdir -p /var/log/journal
 
-exec /usr/local/bin/fluentd $FLUENTD_ARGS
+exec /usr/local/bin/fluentd "${FLUENTD_ARGS}"
```

Comment:
Is there a reason we need to use exec?

Reply:
I think this should be correct, because after we modify FLUENTD_ARGS, the verify-shellcheck.sh check pass.
Before modification:
```
In ./cluster/addons/fluentd-elasticsearch/fluentd-es-image/run.sh line 23:
exec /usr/local/bin/fluentd $FLUENTD_ARGS
                            ^-----------^ SC2086: Double quote to prevent globbing and word splitting.
```

Reply:
fluentd will gracefully stop (flush buffers, etc) if it receives the signal directly.  I assume (not sure!  would love an answer or a bit of a doc/code pointer!) that the kubelet sends a `docker stop` or CRI equivalent to the pod's containers when the pod is being shut down.  

if exec is not used, fluentd isn't PID 1 and doesn't get the signal.